text_processing/RegExpPart1.py

```diff
@@ -53,9 +53,6 @@
 x = re.match("cat", "A cat and a rat can't be friend.")
 print(x)
 
-# will match from start of string
```

Comment:
Not pay attention to this. I changed the code with last time's. And you added them during the meeting, maybe this is the reason.